### PR TITLE
Admin view feature

### PR DESCRIPTION
**Pull request by Index01**
- frontend exists with /admin wireframe
    - backend initial freatures, view exists
    - model updated to include staff
    - initial pop_res.py functionality forklifted to /views/admin.py
    - backend and frontend startup from make
    - end to end verified admin login to guest load

### CODE REVIEW

**Comment by Index01** [comment]
I think this is a good MVP
would merge

**Review by mpesavento — APPROVED** [review]
Looks good, it works!

TODO:
* add frontend feedback (spinning wheel) while backend is running through the load Guests
* fix sign in formatting
* add upload file button instead of reading from package source for guest updates
* reports and stuph

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,6 @@
 frontend_build:
 	docker build -t roombaht:latest frontend/
 	docker run -u node \
-		-e REACT_APP_API_ENDPOINT=$(shell ./scripts/secrets show REACT_APP_API_ENDPOINT) \
```
[the remark this answers is not in the document]
Comment (Index01) [reply Makefile:4]: coooorreect. is set in the Dockerfile. was conflicting here

File: backend/reservations/models.py [hunk 2]
```diff
@@ -12,6 +11,15 @@ class Guest(models.Model):
     def __str__(self):
         return self.name
 
+class Staff(models.Model):
+    name = models.CharField("Name", max_length=240)
+    email = models.EmailField()
+    is_admin = models.BooleanField("Admin", default=False)
```
[the remark this answers is not in the document]
Comment (Index01) [reply backend/reservations/models.py:17]: is a table of "Staff", some of of those staff members happen to be admins. enables things like room assignment, as a staff member could assign rooms, without being able to perform roombot admin things. 
guest field is to associate with a guest entry that has a jwt, that way we can use the same login for both admin purposes and managing our own rooms. especially useful for testing when we want to assign 3-4 rooms to ourselves, swap them around, make sure everything is working - and also use the same loging for admin stuff

**Pull request merged**